Makefile

```diff
@@ -147,6 +148,14 @@ check: ## Run basic go tests with coverage output
 check: get-pgmonitor
 	QUERIES_CONFIG_DIR="$(CURDIR)/${QUERIES_CONFIG_DIR}" $(GO_TEST) -cover ./...
 
+# Informational only; no criteria to enforce at this time.
+.PHONY: check-crd
+check-crd:
+	$(foreach CRD, $(wildcard config/crd/bases/*.yaml), \
+		$(CRD_CHECKER) check-manifests --disabled-validators 'NoBools,NoMaps' --new-crd-filename '$(CRD)' 2>&1 | \
```

Comment:
Do we need a note about why we're turning off these validators?

Reply:
🤔 There was a lot of noise from upstream types, but that can be something we decide later. I'll take this arg out.